src/sidebars/sidebar.less

```diff
@@ -35,6 +35,16 @@
   -webkit-animation-name: favourites;
 }
 
+.version {
+  bottom: 15px;
+  left: 70px;
+  position: absolute;
+  color: @icon-colour;
+  font-weight: @font-light;
+  font-size: 11px;
+  cursor: default;
+}
```

Comment:
It's a very minor fix to take it out or change it if he doesn't like it. So I'd say merge first, ask questions later in this case